pkg/monitortests/network/disruptionpodnetwork/monitortest.go

```diff
@@ -179,8 +182,23 @@ func (pna *podNetworkAvalibility) StartCollection(ctx context.Context, adminREST
 		return err
 	}
 
-	if _, err := pna.kubeClient.CoreV1().Services(pna.namespaceName).Create(context.Background(), hostNetworkTargetService, metav1.CreateOptions{}); err != nil {
-		return err
+	// disable the poller until can get the image sorted out.
+	// TODO re-enable
+	if false {
```

Comment:
The image issues are fixed.  Do it like the other pollers.

Reply:
done